docker-compose.yml

```diff
@@ -48,8 +48,12 @@ services:
       MONGODB_URL: mongodb://mongodb:27017
       SERVER_SHOW_ERRORS: "true"
       SRV_HORIZON_URL: http://mock-horizon:3000
-      SRV_NOTIFY_URL: http://mock-notify:3000
+      SRV_NOTIFY_BASE_URL: http://mock-notify:3000
+      SRV_NOTIFY_SERVICE_ID: 'dummy-service-id-for-notify'
+      SRV_NOTIFY_API_KEY:
+      SRV_NOTIFY_TEMPLATE_ID: '15ed37a9-506c-4845-88ea-95502282a863'
       SRV_OS_PLACES_URL: http://mock-os-places:3000
+
```

Comment:
Can you remove this line? It at least keeps it obvious that it's part of the same block

Reply:
OK